service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -84,8 +65,8 @@ public DatasetAccessLevel getRole(AuthenticatedUserRequest user, String workspac
     }
   }
 
-  WorkspacesApi workspacesApi(AuthenticatedUserRequest user) {
-    return new WorkspacesApi(getApiClient(user));
+  private WorkspacesApi workspacesApi(AuthenticatedUserRequest user) {
```

Comment:
Same as the comment in DatarepoService. My inclination is to inline this, but I am okay with this approach. However, we should make sure all our services are taking the same approach.